docs/man/Makefile.am

```diff
@@ -242,6 +246,9 @@ MAN3_DEV_PAGES = \
 	nutscan_get_serial_ports_list.3 \
 	nutscan_init.3
 
+$(LIBNUTCLIENT_MISC_DEPS): libnutclient_misc.3:
```

Comment:
Is the trailing colon `:` correct here (I don't remember seeing such syntax; if it exists - is it portable to non-GNU `make`s)?

Also, doesn't this recipe define that each of those six files (in LVAL) depend on this one (RVAL) and after this prerequisite is met (RVALwas generated), the rule to make the LVALs is to just touch (thus creating just empty files)? Or is this what the trailing colon fixes somehow? Possibly, the correct rule might be to `$(MAKE) $@` those targets...

DISCLAIMER: I did not (and in the near future likely won't) look into the full context code of the Makefile, so quite likely I am missing something.